src/firefly/java/edu/caltech/ipac/firefly/data/table/MetaConst.java

```diff
@@ -15,21 +15,34 @@
  */
 public class MetaConst {
 
+    /**
+     * Meta entry that defines how to compute the center position of table row
+     * Form: lonCol;latCol;CoordinateSys.toString(), eg ra;dec;J2000
+     * @see edu.caltech.ipac.table.TableMeta#setCenterCoordColumns
+     */
     public static final String CENTER_COLUMN = "CENTER_COLUMN";
+
+
+    /**
+     *
+     * Meta entry that defines the corners of a object defined by a table row
+     * like CENTER_COLUMN but each position separated by comma
+     * eg - ra1;dec1;EQ_J2000,ra2;dec2;EQ_J2000,ra3;dec3;EQ_J2000,ra4;dec4;EQ_J2000
+     * @see edu.caltech.ipac.table.TableMeta#setCorners
+     */
     public static final String ALL_CORNERS   = "ALL_CORNERS";
 
-    public static final String WORLD_PT_KEY = "SearchByPosition.worldpt";
-    public static final String RADIUS_KEY = "SearchByPosition.field.radius";
+    /** if defined to any value the the table is a catalog */
     public static final String CATALOG_OVERLAY_TYPE = "CatalogOverlayType";
```

Comment:
This is not quite true, right?
It looks like this is only used to determine whether `guessing` is allowed when searching for center columns.  As long as there's center columns info, it's a catalog regardless of this setting.

Reply:
I do need to make it more clear.  `CatalogOverlayType` is required to even guess if there is not VO information. Also it can be defined to FALSE to disable it being a catalog altogether even with VO information.

The problem is then we could guess it to be a catalog and it could be image meta data with corner information.  Therefore `CatalogOverlayType` is required to treat it as a catalog.

I have spent some time thinking about this and I don't see anyway we can deal with simple tables like cvs, tsv, ipac, or FITS with something telling us how to treat the table.

Reply:
Ah.. it makes sense now.  Too bad prop is already used by others.  Otherwise, having a prop like `data_type` to say what this table is('image-meta'), instead of using `CatalogOverlayType=false` to say what it is not would make more sense.